skfp/datasets/tdc/tox/dili.py

```diff
@@ -23,14 +23,28 @@ def load_dili(
 ) -> Union[pd.DataFrame, tuple[list[str]], np.ndarray]:
     """
 
+
+    Load and return the DILI (Drug Induced Liver Injury) dataset from TDC benchmark.
+
+    The task is to predict whether the drugs can cause liver injury given a set of molecules [1]_.
```

Comment:
TDC has nice description, follow it more. Also, remove "given a set of molecules", it's obvious from context

Reply:
And cite TDC